org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ClassActionImpl.java

```diff
@@ -950,6 +950,9 @@ private Object transformConstantValue(Object inputValue) {
 				if (outputString == null) {
 					transformCase = "direct";
 					outputString = transformDirectString(inputString);
+                                        if (outputString == null) {
```

Comment:
Add a "transformCase" tag for this case, for example, "direct per class".

Reply:
Thanks, done.